ts/src/core/parser/index.ts

```
@@ -3,43 +1,138 @@
+              const events = await parseJSONLFile(filePath, startTime, endTime);
+              allEvents.push(...events);
+            }
+          }
+        } catch (error) {
+          // Skip inaccessible directories
+          continue;
         }
       }
+    } catch (error) {
+      // Directory doesn't exist, try the next one
+      continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)